processing/db_handler.py

```diff
@@ -196,6 +197,181 @@ def get_filename_checksum_lookup(self, conn):
                 checksums[fname] = r.get('file_checksum')
         return checksums
 
+    def get_data_upload_log_with_uuid(self,
+                                      conn,
+                                      uuid):
+        query = SQL('SELECT u.log_id, u.site_id, '
+                    'u.data_file, u.upload_token, '
+                    'u.upload_comment, u.upload_type_id '
+                    'FROM input_interface.data_upload_log u '
+                    'LEFT JOIN '
+                    'input_interface.data_upload_file_xfer_log x '
+                    'ON u.log_id = x.upload_log_id '
+                    'WHERE '
+                    'u.upload_type_id IN (4, 7) '
```

Comment:
Previously, I made a comment about this. Not sure if we are keeping as is or moving towards using the text description